index.html

```diff
@@ -41,6 +40,151 @@ <h1>I'm Tash <br> Glad to see you!</h1>
 			    </div>
 			</div>
 		</section>
+
+		<section class="works">
+			<div class="works1">
+			<img class="snap" src="images/Snap3.png" alt="">
+			<p class="tonic">Tonic</p>
+
+			<div class="grid-container">
+				  <div class="grid-item1"><p>CANOPY</p></div>
+				  <div class="grid-item2"><p>.</p></div>
+				  <div class="grid-item3"><p>Back End Dev</p></div>
+				  <div class="grid-item4"><p>.</p></div>
+				  <div class="grid-item5"><p>2015</p></div>
+			</div>
+			<p class="day">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs">
+				<button class="btn1" type="submit">html</button>
+				<button class="btn2" type="submit">css</button>
+				<button class="btn3" type="submit">javaScript</button>
+			</div>
+			<button class="btn4" type="submit">See Project</button>
+			</div>
+
+			<div class="works2">
+			<img class="snap2" src="images/Snap4.png" alt="">
+			<p class="tonic2">Multi-Post Stories</p>
+
+			<div class="grid-container2">
+				  <div class="grid-item12"><p>CANOPY</p></div>
+				  <div class="grid-item22"><p>.</p></div>
+				  <div class="grid-item32"><p>Back End Dev</p></div>
+				  <div class="grid-item42"><p>.</p></div>
+				  <div class="grid-item52"><p>2015</p></div>
+			</div>
+			<p class="day2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs2">
+				<button class="btn12" type="submit">html</button>
+				<button class="btn22" type="submit">css</button>
+				<button class="btn32" type="submit">javaScript</button>
+			</div>
```

Comment:
- [ ] For the list of technologies, it would be better to use a `ul` tag instead of a `div` since they're basically a list of technologies. This will improve the semantic structure of your website. Also, the technologies are not supposed to be pressable and interactive buttons so using a `li` tag would be a better representation.